### PR TITLE
[TASK] Added README.md to project

### PR DESCRIPTION
**Pull request by sk-sparhandy**
Resolves: https://github.com/Sparhandy/CodingStandard/issues/4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Build Status of branch master [![Build Status](https://travis-ci.org/Sparhandy/CodingStandard.svg?branch=master)](https://travis-ci.org/Sparhandy/CodingStandard)
```
[the remark this answers is not in the document]
Comment (PhilippWitzmann) [reply README.md]: [Packagist]()

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```bash
+composer require "sparhandy/codingstandard" "dev-master"
+```
+
+## Usage
+
+### From Terminal (Cli)
+
+To execute PHP Code Sniffer configured for sparhandy coding standards do the following.
```
Comment (PhilippWitzmann) [inline README.md]: #### PHP Code Sniffer

File: README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```bash
+./vendor/bin/phpcs --extensions=php --standard=./vendor/sparhandy/codingstandard/src/phpcs/Production/ruleset.xml ./path/to ./your/sources 
+```
+
```
Comment (PhilippWitzmann) [inline README.md]: #### PHP Mess Detector

File: README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+To execute PHP Code Sniffer configured for sparhandy coding standards do the following.
```
Comment (PhilippWitzmann) [inline README.md]: ~~PHP Code Sniffer~~ PHP Mess Detector

File: README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+* Create a fork of Sparhandy/CodingStandard.
+* Create your branch from master and commit your changes.
+* Push your branch to your fork.
+* Create a pull request on GitHub.
+* Discuss your pull request with us.
+* Our devs will then merge or close the PR.
+
+# Acknowledgements
+This would not be possible without the tremendous work of the good people at [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) and [PHP Mess Detector](https://github.com/phpmd/phpmd) and of cause the authors annotated in the source files.
```
Comment (PhilippWitzmann) [inline README.md]: ~~of cause~~ of course

File: README.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+# How to contribute
+
+If you want to contribute to the standard here is how it works.
+
+* Create a fork of Sparhandy/CodingStandard.
+* Create your branch from master and commit your changes.
+* Push your branch to your fork.
+* Create a pull request on GitHub.
+* Discuss your pull request with us.
+* Our devs will then merge or close the PR.
```
Comment (PhilippWitzmann) [inline README.md]: Please use pull requets instead of PR

Comment (PhilippWitzmann) [inline README.md]: How about
"Our devs will either merge or decline the pull request"

File: README.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+# Acknowledgements
+This would not be possible without the tremendous work of the good people at [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) and [PHP Mess Detector](https://github.com/phpmd/phpmd) and of cause the authors annotated in the source files.
+
```
Comment (PhilippWitzmann) [inline README.md]: Please remove the empty line

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sk-sparhandy) [reply README.md]: Will do

File: README.md (hunk 2 above)
Comment (sk-sparhandy) [reply README.md]: Will do

File: README.md (hunk 3 above)
Comment (sk-sparhandy) [reply README.md]: will do

File: README.md (hunk 4 above)
Comment (sk-sparhandy) [reply README.md]: Will do

File: README.md (hunk 6 above)
Comment (sk-sparhandy) [reply README.md]: will do

File: README.md (hunk 5 above)
Comment (sk-sparhandy) [reply README.md]: will do

File: README.md (hunk 7 above)
Comment (sk-sparhandy) [reply README.md]: Will do

**Pull request merged**